server/pulp/server/webservices/views/consumers.py

```diff
@@ -198,6 +276,42 @@ def put(self, request, consumer_id):
         return generate_json_response_with_pulp_encoder(consumer)
 
 
+class ConsumerSearchView(search.SearchView):
+    """
+    This view provides GET and POST searching for Consumers.
+    """
+    optional_fields = ['details', 'bindings']
+    response_builder = staticmethod(generate_json_response_with_pulp_encoder)
```

Comment:
I think they do all currently use "with_pulp_encoder", but I also suspect that will change if we change the way we serialize dates (which I think we probably should). Leaving as is will make that an easier transition.